Source/EasyNetQ/Consumer/IConsumerDispatcher.cs

```diff
@@ -4,7 +4,18 @@ namespace EasyNetQ.Consumer
 {
     public interface IConsumerDispatcher : IDisposable
     {
-        void QueueAction(Action action);
```

Comment:
Breaking change. You could leave `QueueAction` name for transient case (+ of course documenting this) and add only new `QueueDurableAction` method.

Reply:
I personally like your option too.

Reply:
I am up for both. I prefer breaking the interface on this one - to make it very clear what you are doing. @zidad what is you oppinion on this ? :)

Reply:
I’ll just remind that this is still a breaking change - the calling code must be recompiled.

Reply:
True i will add the original method to the interface 👌☺️

Reply:
I personally do not really care 😄